Sources/Datadog/CrashReporting/CrashReportingFeature.swift

```diff
@@ -33,15 +33,19 @@ internal final class CrashReportingFeature {
 
     init(
         configuration: FeaturesConfiguration.CrashReporting,
-        commonDependencies: FeaturesCommonDependencies
+        consentProvider: ConsentProvider,
+        userInfoProvider: UserInfoProvider,
+        networkConnectionInfoProvider: NetworkConnectionInfoProviderType,
+        carrierInfoProvider: CarrierInfoProviderType,
+        appStateListener: AppStateListening
     ) {
```

Comment:
Yes, mainly because crash reporting integrations are expecting a `DatadogV1Context` at initialisation. The plan is to replace these integrations with the message bus.

Reply:
👌 great!